### PR TITLE
Viewer delete<index> 

### LINKED ISSUE #13: Viewer delete<index>
Deletes a PC from the PCList

### PR DESCRIPTION
**Pull request by Reynold-SL**
Create ViewerDeleteCommand to delete PC from the PC list

### CODE REVIEW

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import seedu.bigpp.pc.PCList;
+
+public class ViewerDeleteCommand extends Command {
+
+    public ViewerDeleteCommand(String arguments) {
+        super.setArguments(arguments);
+    }
+
+    /**
+     * Delete a PC of a given index from the PC list
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java:14]: Good practice to add documentation to new methods

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+     *
+     * @return if the index is a Prebuilt PC, inform the user that the PC of that index cannot be deleted
+     * @return the name of the PC deleted
+     */
+    @Override
+    public String executeCommand() {
+        int index = Integer.parseInt(super.getArguments()) - 1;
+        PC pc = (PCList.getList()).get(index);
+        if (pc.getIsPreBuilt()) {
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java:23]: Good handling of case where user is trying to delete a prebuilt pc(cannot be changed or deleted)

**Review by Ryujikjs — COMMENTED** [review]
Documentation of code is clear and allows for better understanding of method.

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import seedu.bigpp.command.Command;
+import seedu.bigpp.pc.PC;
+import seedu.bigpp.pc.PCList;
+
+public class ViewerDeleteCommand extends Command {
+
+    public ViewerDeleteCommand(String arguments) {
+        super.setArguments(arguments);
+    }
```
Comment (typingpanda) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java:11]: clear naming of methods which follows the coding standards

**Review by typingpanda — COMMENTED** [review]
LGTM

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java [hunk 4]
```diff
@@ -10,6 +10,10 @@ public ViewerAddCommand(String arguments) {
         super.setArguments(arguments);
     }
 
+    /**
+     * Add new PC of a given name to the PC list
+     * @return the name of the PC created
+     */
```
Comment (typingpanda) [inline src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java:16]: awesome documentation

**Review by typingpanda — APPROVED** [review]
Well done! Code is ready to merge with master

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java (hunk 1 above)
Comment (Reynold-SL) [reply src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java:14]: Will use this format for future documentation

File: src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java (hunk 2 above)
Comment (Reynold-SL) [reply src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java:22]: Keep this structure for this method in future implementations

**Pull request merged**